src/Jackett.Common/Indexers/Abstract/AvistazTracker.cs

```diff
@@ -106,11 +106,12 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
                 var rows = dom["table:has(thead) > tbody > tr"];
                 foreach (var row in rows)
                 {
-                    CQ qRow = row.Cq();
-                    var release = new ReleaseInfo();
-
-                    release.MinimumRatio = 1;
-                    release.MinimumSeedTime = 172800; // 48 hours
+                    var qRow = row.Cq();
+                    var release = new ReleaseInfo
+                    {
+                        MinimumRatio = 1,
```

Comment:
The only thing I don't like about this PR using the object initializer for 2 parameters and the setters for the others... I like the object initializer if I can set all parameters if not I prefer all setters. What we can do?

Reply:
The automated tool only pulls in initializers if they're immediately after the declaration and in a format that allows them to be used in an initializer. It is entirely possible to make them all one or the other, really no matter which scenario it is, but the code needs to be rearranged to allow it.

This would mean either changing the order of setters so the first one after the object is one that can't be converted to an initializer, or moving some variables above the initialization so they work with initializer format. See my next comment for example.